### ISSUE TITLE
systemd seems to be failing to compile on Debian Stretch

### ISSUE THREAD

**Issue by evverx** [systemd/systemd#13226]
```
[577/1535] Generating random-seed.c.o with a custom command.
FAILED: src/boot/efi/random-seed.c.o
cc -c ../src/boot/efi/random-seed.c -o src/boot/efi/random-seed.c.o -Wall -Wextra -std=gnu90 -nostdinc -ggdb -O0 -fpic -fshort-wchar -ffreestanding -fno-strict-aliasing -fno-stack-protector -Wsign-comparI
../src/boot/efi/random-seed.c:12:43: error: ‘EFI_RNG_PROTOCOL_GUID’ undeclared here (not in a function)
 static const EFI_GUID rng_protocol_guid = EFI_RNG_PROTOCOL_GUID;
                                           ^~~~~~~~~~~~~~~~~~~~~
../src/boot/efi/random-seed.c: In function ‘acquire_rng’:
../src/boot/efi/random-seed.c:19:9: error: unknown type name ‘EFI_RNG_PROTOCOL’
         EFI_RNG_PROTOCOL *rng;
         ^~~~~~~~~~~~~~~~
In file included from /usr/include/efi/efi.h:35:0,
                 from ../src/boot/efi/random-seed.c:1:
../src/boot/efi/random-seed.c:38:36: error: request for member ‘GetRNG’ in something not a structure or union
         err = uefi_call_wrapper(rng->GetRNG, 3, rng, NULL, size, data);
                                    ^
[579/1535] Generating dns_type-from-name.h with a meson_exe.py custom command.
ninja: build stopped: subcommand failed.
Makefile:2: recipe for target 'all' failed
make: *** [all] Error 1
```

**Comment by yuwata** [comment]
Do you have efi/efiprot.h ?

**Comment by evverx** [comment]
```
$ dpkg -S /usr/include/efi/efiprot.h
gnu-efi: /usr/include/efi/efiprot.h

$ dpkg -l gnu-efi
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                                          Version                     Architecture                Description
+++-=============================================-===========================-===========================-===============================================================================================
ii  gnu-efi                                       3.0.4-2                     amd64                       Library for developing EFI applications
```

**Comment by yuwata** [comment]
Fedora 30 has slightly newer one.
```
$ rpm -q --whatprovides /usr/include/efi/efiprot.h
gnu-efi-devel-3.0.8-6.fc30.x86_64
```
And stretch-backport repo has 3.0.9.
https://packages.debian.org/source/stretch-backports/gnu-efi
Is it possible to build systemd with the backported package?

**Comment by evverx** [comment]
As far as I know it's possible to build systemd with `3.0.9-1` shipped on Debian Buster. Looks like we had switched to Buster on Semaphore before https://github.com/systemd/systemd/pull/13137 was merged.

**Comment by yuwata** [comment]
These are added by https://sourceforge.net/p/gnu-efi/code/ci/9b0fe34e208afe9a7e83bb07aa8d7d1b185f15ff/, which is included in 3.0.5.
